include/openenclave/bits/report.h

```diff
@@ -133,6 +133,10 @@ typedef struct _oe_report
 
     /** Contains the IDs and attributes that are part of oe_identity_t */
     oe_identity_t identity;
+
+    /** Contains the result reported by quote verification logic. The size is
```

Comment:
There is a `size` field in this struct. Would the value of that field change with this PR?

Reply:
Yes, but it's always set to sizeof the struct here:https://github.com/openenclave/openenclave/blob/master/common/sgx/report.c#L22